GeneralsMD/Code/GameEngine/Source/GameNetwork/GameInfo.cpp

```diff
@@ -897,7 +897,49 @@ Bool GameInfo::isSandbox(void)
 
 static const char slotListID		= 'S';
 
-AsciiString GameInfoToAsciiString( const GameInfo *game )
+static void BuildPlayerNames(AsciiStringVec& playerNames, const GameInfo& game)
```

Comment:
I suggest build the vector in this class and return it. Because right now this expectes passing the vector with the correct size which is not so nice.